multiqc/modules/bclconvert/bclconvert.py

```diff
@@ -504,29 +511,27 @@ def _split_data_by_lane_and_sample(self, bclconvert_data):
                         bclconvert_by_sample[sample_id] = self._reads_dictionary()
 
                     s = bclconvert_by_sample[sample_id]
-
                     s["clusters"] += int(sample["clusters"])
                     s["yield"] += int(sample["yield"])
                     s["perfect_index_reads"] += int(sample["perfect_index_reads"])
                     s["one_mismatch_index_reads"] += int(sample["one_mismatch_index_reads"])
                     s["basesQ30"] += int(sample["basesQ30"])
-                    s["mean_quality"] += float(sample["mean_quality"])
                     s["cluster_length"] = lane["cluster_length"]
+                    s["_quality_score_sum"] += sample["_quality_score_sum"]
```

Comment:
If this is an internal variable, should check whether it now makes it through to the `multiqc_data` exports. Might need deleting again before being saved there if so.